connect_bi_reporter/feeds/api/schemas.py

```diff
@@ -0,0 +1,25 @@
+from typing import Optional
+
+from connect_bi_reporter.schemas import Events, NonNullSchema, ReferenceSchema
+
+
+class ScheduleReportReferenceSchema(NonNullSchema):
```

Comment:
[COD-C] We could use the ReferenceSchema for this and Credential, the only attribute mandatory is the id.

Reply:
updated!